samples/iolib/src/main/cpp/player/SimpleMultiPlayer.h

```diff
@@ -22,24 +22,22 @@
 #include "OneShotSampleSource.h"
 #include "SampleBuffer.h"
 
-using namespace oboe;
-
 namespace iolib {
 
 typedef unsigned char byte;     // an 8-bit unsigned value
 
 /**
  * A simple streaming player for multiple SampleBuffers.
  */
-class SimpleMultiPlayer : public AudioStreamCallback  {
+class SimpleMultiPlayer : public oboe::AudioStreamCallback  {
 public:
     SimpleMultiPlayer();
 
     // Inherited from oboe::AudioStreamCallback
-    DataCallbackResult onAudioReady(AudioStream *oboeStream, void *audioData,
+    oboe::DataCallbackResult onAudioReady(oboe::AudioStream *oboeStream, void *audioData,
             int32_t numFrames) override;
-    virtual void onErrorAfterClose(AudioStream *oboeStream, Result error) override;
-    virtual void onErrorBeforeClose(AudioStream * oboeStream, Result error) override;
+    virtual void onErrorAfterClose(oboe::AudioStream *oboeStream, oboe::Result error) override;
+    virtual void onErrorBeforeClose(oboe::AudioStream * oboeStream, oboe::Result error) override;
 
     void setupAudioStream(int32_t numSampleBuffers, int32_t channelCount, int32_t sampleRate);
```

Comment:
Does `SimpleMultiPlayer` make sense without an audio stream? If not, then move these parameters into the constructor and initialize them there.

Reply:
It does because the stream will get allocated/deallocated (created/destroyed) separately from the SampleBuffer/Sample source pairs based on application life-cycle.